kafka-ui-api/src/main/java/com/provectus/kafka/ui/config/Constants.java

```diff
@@ -0,0 +1,18 @@
+package com.provectus.kafka.ui.config;
+
+public class Constants {
+
+  public static final String[] AUTH_WHITELIST = {
```

Comment:
I suggest to move it to auth related class since it is auth-related config (maybe create AbstractAuthSecurityConfig?)

Reply:
also, since we dont support auth.enabled property now - I think we need to print error messages / fail on startup

Reply:
Regarding `auth.enabled`: @germanosin suggested we don't stop the app because of that, that's why I just left the `auth.type` defaulted to LOGIN_FORM to keep it secure.
Regarding auth-related config: done